python/BioSimSpace/Process/_atom_utils.py

```diff
@@ -0,0 +1,977 @@
+######################################################################
```

Comment:
Can this file be call `_atm_utils.py` to match the other modules.

Reply:
Now changed (https://github.com/OpenBioSim/biosimspace/blob/feature_AToM/python/BioSimSpace/Process/_atm_utils.py)